src/codeu/chat/client/ClientUser.java

```diff
@@ -109,6 +114,18 @@ public void showAllUsers() {
     }
   }
 
+  public User searchByName(String s) {
+    updateUsers();
+    for (final User u : usersByName.all()) {
+      if (s.equals(u.name)){
+	return u; 
+    }
+   }
+	return null;
```

Comment:
Tabbing (and now that I look at it, spacing in general) is weird in this function. 1 space indents?